spec.html

```diff
@@ -36211,7 +36211,7 @@ <h1>The DataView Constructor</h1>
 
       <!-- es6num="24.2.2.1" -->
       <emu-clause id="sec-dataview-buffer-byteoffset-bytelength">
-        <h1>DataView ( _buffer_ [ , _byteOffset_ [ , _byteLength_  ] ] )</h1>
+        <h1>DataView ( _buffer_, _byteOffset_, _byteLength_ )</h1>
```

Comment:
Whether we decide on 1 or 3, this signature shouldn't be changed; if we go with 3, then like other legacy/deviating lengths, let's use a note.